src/parser/parsers/changelogs.py

```diff
@@ -11,27 +13,60 @@
 from .constants import OUTPUT_DIR
 
 
+
+class HTMLTagRemover(HTMLParser):
+    # https://www.slingacademy.com/article/python-ways-to-remove-html-tags-from-a-string#Using_HTMLParser
+    def __init__(self):
+        super().__init__()
+        self.result = []
+
+    def handle_data(self, data):
+        self.result.append(data)
+
+    def get_text(self):
+        return ''.join(self.result)
+
+
 class ChangelogParser:
     def __init__(self):
         self.CHANGELOGS_DIR = os.path.join(os.path.dirname(__file__), '../raw-changelogs/')
         self.OUTPUT_DIR = OUTPUT_DIR
-        self.OUTPUT_CHANGELOGS = self.OUTPUT_DIR+'/changelogs'
+        self.OUTPUT_CHANGELOGS = self.OUTPUT_DIR + '/changelogs'
         self.resources = self._get_resources()
 
-    def run_all(self):
-        changelogs_by_date = {}
+    # download rss feed from changelog forum and parse entries
+    def fetch_forum_changelogs(self):
```

Comment:
this whole fetching process feels like it could be its own file and class. Thoughts on a `parsers/changelog` folder which contains `changelog_parser.py` and `fetch_changelogs.py`?

Reply:
Separated them!  Now there's a `parse_changelogs` and a `fetch_changelogs`.  `fetch_*` gathers the txt and rss stuff, and prepares the logs.  It will load logs from rss, then the txt files.  With any duplicate changelog titles, the txt logs will be used instead of the fetched one.  This will make it so we can still customize the changelog text in case manual editing is required for some reason